template.rb

```diff
@@ -210,6 +210,20 @@ def add_esbuild_script
   end
 end
 
+def add_github_actions_ci
+  script = <<~EOS
+    # See https://github.com/andyw8/setup-rails for more information
+  
+    name: Verify
+    on: [push]
+
+    jobs:
+      verify:
+        uses: andyw8/setup-rails/.github/workflows/verify.yml@v1
+  EOS
+  create_file '.github/workflows/verify.yml', script
+end
```

Comment:
Could we use `copy_file` instead for this? 

Might conflict with the repo's .github folder, so we might want to put a `github/workflows/verify.yml` instead and specify the location: https://rubydoc.info/gems/thor/0.15.4/Thor%2FActions:copy_file